Makefile

```diff
@@ -113,6 +113,8 @@ INCDIR      := include
 L3_SRCDIR   := $(SRCDIR)
 L3_INCDIR   := $(INCDIR)
 USE_CASES   := use-cases
+L3_UTILS    := utils
+L3_UTILSDIR := $(USE_CASES)/$(L3_UTILS)
```

Comment:
Am adding the new helper functions under `use-cases/utils/` as I next plan to use the `size_str()` interfaces in the `use-cases/client-server-msgs-perf` programs.

These helper conversion methods are not core-L3 functionality, hence, rather than littering a top-level `utils/` sub-dir, I chose to stash them under the `use-cases/utils/` dir.

If we do find more wide-spread for these helper conversion functions, we can look at relocating the sources at some more visible upper-level dirs.